docs/assets/templates/notebooks/template-big.dockstore.yml

```diff
@@ -0,0 +1,74 @@
+# The first line refers to the version 1.2 of the .dockstore.yml schema
+version: 1.2
+
+# An array of notebooks. Each element corresponds to a notebook on Dockstore.
+notebooks:
+  
+  # The optional notebook name for a notebook, which may only consist of alphanumerics
+  # and internal underscores and hyphens, but no spaces or other characters. Names may not exceed 256 characters.
+  # If using a .dockstore.yml with multiple notebooks, this field is required 
+  # to uniquely identify notebooks in the repository.
+  #
+  # It should be noted that having the name come first is an arbitrary decision.
+  # You could use subclass instead, for instance. Provided arrays are not broken
+  # up, the order of fields within a .dockstore.yml is not important.
+  - name: <String>
+    
+    # The format used for the notebook. Currently, only JUPYTER is supported.
+    # This cannot be changed once the notebook is registered.
+    format: <JUPYTER>
+
+    # The programming language of the code contained in the notebook.
+    language: <PYTHON | R | JAVASCRIPT | SCALA | JULIA | OTHER>
+
+    # The absolute path to the notebook file in the Git repository.
+    # - For Jupyter notebooks, the notebook is a .ipynb file.
+    path: <String>
+
+    # The kernel image reference for the notebook.
+    kernel: <String>
+    
+    # Notebook-wide setting that will affect ALL branches/tags; only set this as needed in a main branch.
+    # Set to true to publish an unpublished notebook, or false to unpublish a published notebook.
+    # Omitting the publish setting leaves the publish-state unchanged (recommended for all non-primary branches).
+    publish: <Boolean>
+
+    # An optional array of absolute paths to other files in the Git repository that should be read and included with the notebook.
+    otherFiles: <String Array>
+
+    # An optional array of absolute paths to test parameter files in the Git repository.
+    # For example...
+    # testParameterFiles:
+    #   - /null-model/null-model.json
+    #   - /null-model/null-model-binary.json
+    testParameterFiles: <String Array>
+
+    # An optional path to a notebook-specific readme in the Git repository. If not provided, Dockstore will show
+    # the readme.md present at the root of the Git repository if it is present.
+    # If you have multiple notebooks in a single Git repository, it is recommend to give each one a readme.
+    readMePath: <String>
+
+    # An optional array of authorship information.
+    # Note that if orcid is present, then all other fields will be ignored, as information will be taken from orcid.
+    # If orcid is not present, make sure to at a minimum include the name field for each author.
+    authors:
+      - name: <String>
```

Comment:
Maybe re-order to emphasize that orcid is preferred/easier since it is automatically kept up-to-date with info from orcid